### PR TITLE
Koolis 6by6 lugu infidels

### PR DESCRIPTION
**Pull request by L0rdMortimer**

### CODE REVIEW

File: 05-exxxxx/05-Exxxs/Program.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ static void Main(string[] args)
 
 
             }
+            Console.WriteLine("hub hub hub. infidels");
```
Comment (RaitVeld) [inline 05-exxxxx/05-Exxxs/Program.cs]: What this infidels do here?
Delete it

File: 05-exxxxx/05-Exxxs/Program.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace _05_Exxxs
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+;            for (int n = 1; n < 6; n++)
+            {
+                Console.Write("{0}\t",n);
```
Comment (RaitVeld) [inline 05-exxxxx/05-Exxxs/Program.cs:15]: Mida see {0} teeb?

**Review by RaitVeld — COMMENTED** [review]
Comment

File: 05-exxxxx/05-Exxxs/Program.cs (hunk 2 above)
Comment (L0rdMortimer) [reply 05-exxxxx/05-Exxxs/Program.cs:15]: pane käima saad teada.

**Pull request merged**